### ISSUE TITLE
Fetching only specific fields from a collection list

### ISSUE THREAD

**Issue by saginadir** [pocketbase/pocketbase#2354]
I would like to use this opportunity to say that I really appreciate the effort put into this project. It's a great solution all around.

To my issue,

Each row of my collection is storing 1MB of data, but this data is mainly on a single column. For example my collection can be something like an article:

```JSON
{
  "title": "hello world",
  "body": "This is a very long body that can be about 1MB in size... ..."
}
```

is there a way to fetch the `title` without the `body` using the js sdk? I dug around and couldn't find any viable example.

Thanks for the help!

**Comment by ganigeorgiev** [comment]
This is already in the [roadmap](https://github.com/orgs/pocketbase/projects/2?pane=issue&itemId=16153865) and will be implemented in the near future (either in v0.16 or v0.17 release; there are no ETAs yet).

To some extend, a workaround could be to create a [View collection](https://pocketbase.io/docs/collections/#view-collection) with subset of the fields you want to export (the minimum is to have an `id`).

**Comment by ganigeorgiev** [comment]
_Just for info, support for `?fields=field1,field2` query parameter to limit the returned API fields was added in the develop branch and will be available with the next v0.16.0 release (there are no ETAs yet)._
